frontend/components/exercise/FillInTheBlank.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+
+interface FillInTheBlankProps {
```

Comment:
also i had already made ConversationBubble. if yours is better then fine but can you double check lol. if so then we can delete conversation bubble. although it was nice to have each bubble as a component, see if there's a way you can incorporate it? or idk